torch-neuronx/training/tp_dp_gpt_neox_20b_hf_pretrain/modeling_gpt_neox_nxd.py

```diff
@@ -0,0 +1,818 @@
+# coding=utf-8
```

Comment:
Overall can we avoid copying the entire file and override functions whereever needed? This would make it easier to upstream too and also help users to understand what changes are needed to convert their existing models to use TP+SP

Reply:
Have cleaned up the code by deleting unnecessary codes.